core/storage/collection/gae_models_test.py

```diff
@@ -0,0 +1,208 @@
+                version=0,
+                node_count=0,
+                collection_model_last_updated=None,
+                collection_model_created_on=None,
+            ))
+        public_collection_summary_model.put()
+
+        private_collection_summary_model = (
+            collection_models.CollectionSummaryModel(
+                id='id1',
+                title='title',
+                category='category',
+                objective='objective',
+                language_code='language_code',
+                tags=['tags'],
+                status=constants.ACTIVITY_STATUS_PRIVATE,
+                community_owned=False,
+                owner_ids=['owner_ids'],
+                editor_ids=['editor_ids'],
+                viewer_ids=['viewer_ids'],
+                contributor_ids=[''],
+                contributors_summary={},
+                version=0,
+                node_count=0,
+                collection_model_last_updated=None,
+                collection_model_created_on=None,
+            ))
+        private_collection_summary_model.put()
+        commits = collection_models.CollectionSummaryModel.get_non_private()
+        self.assertEqual(1, len(commits))
+
+    def test_get_private_at_least_viewable(self):
+        viewable_collection_summary_model = (
+            collection_models.CollectionSummaryModel(
+                id='id0',
+                title='title',
+                category='category',
+                objective='objective',
+                language_code='language_code',
+                tags=['tags'],
+                status=constants.ACTIVITY_STATUS_PRIVATE,
+                community_owned=False,
+                owner_ids=['owner_ids'],
+                editor_ids=['editor_ids'],
+                viewer_ids=['a'],
+                contributor_ids=[''],
+                contributors_summary={},
+                version=0,
+                node_count=0,
+                collection_model_last_updated=None,
+                collection_model_created_on=None,
+            ))
+        viewable_collection_summary_model.put()
+
+        unviewable_collection_summary_model = (
+            collection_models.CollectionSummaryModel(
+                id='id1',
+                title='title',
+                category='category',
+                objective='objective',
+                language_code='language_code',
+                tags=['tags'],
+                status=constants.ACTIVITY_STATUS_PRIVATE,
+                community_owned=False,
+                owner_ids=['owner_ids'],
+                editor_ids=['editor_ids'],
+                viewer_ids=['viewer_ids'],
+                contributor_ids=[''],
+                contributors_summary={},
+                version=0,
+                node_count=0,
+                collection_model_last_updated=None,
+                collection_model_created_on=None,
+            ))
+        unviewable_collection_summary_model.put()
+        commits = (
```

Comment:
why is this defined as commits? I think this is going to be a list of collection models, right? (ditto above)

Reply:
Done. Didn't change it from the copy paste.